docs/1.0. Certificate Provisioning.md

```diff
@@ -231,9 +234,17 @@ For each generated CSR the EST Client SHOULD make a HTTPS request containing the
 
 #### Certificate Request Response
 
-If the EST Server returns a HTTP 200 response the certificate request was successful and the EST Client should use the returned TLS Certificate and its chain of trust for all further requests to NMOS APIs. The EST Client MUST remove any previously issued TLS Certificates and keys.
+If the EST Server returns a HTTP 200 response the certificate request was successful and the response from the EST Server SHOULD contain a valid TLS certificate.
+
+Before using the returned TLS certificate the EST Client SHOULD validate the TLS certificate, validating for example the; NotBefore, Not After, Common Name, Subject Alt Name and chain of trust. The validity of the TLS certificate could checked by installing the certificate and performing a request to the API.
```

Comment:
```suggestion
Before using the returned TLS certificate the EST Client SHOULD validate the TLS certificate, validating for example the; NotBefore, Not After, Common Name, Subject Alt Name and chain of trust. The validity of the TLS certificate could be checked by installing the certificate and performing a request to the API.
```

Reply:
Perhaps a short sentence needs adding for if validation fails, and how fallback should be handled?